routes/handlers/facebook.js

```diff
@@ -1,9 +1,14 @@
 const Facebook = require('../../lib/facebook');
 
 module.exports.events = {
-  handler: function (request, reply) {
-    Facebook.getEvents(function(err, res) {
-      return reply(res).type('application/json');
-    });
-  }
+    handler: function (request, reply) {
+        Facebook.getEvents(function (err, res) {
+            return reply(res);
+        });
+    },
+    plugins: {
+        hal: {
```

Comment:
Do we have a plugin for hal? Did you mean to check this in? 

I added OSDI support below.
https://github.com/ResistanceCalendar/resistance-calendar-api/pull/10